docs/fedora-releases-guide.md

```diff
@@ -144,6 +144,10 @@ via a comment in any dist-git pull request:
 
     /packit pull-from-upstream
 
+This will take the `.packit.yaml` configuration file from the `rawhide` dist-git branch, same as if the job was triggered by a new release. To use configuration file from the dist-git pull request you are commenting on, you can add an argument:
```

Comment:
```suggestion
This will take the Packit configuration file from the `rawhide` dist-git branch, same as if the job was triggered by a new release. To use the configuration file from the dist-git pull request you are commenting on, you can add an argument:
```